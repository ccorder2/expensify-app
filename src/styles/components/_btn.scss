.btn {
	background: $blue;
	border: none;
	color: white;
	display: inline-block;
	font-size: $l-font-size;
	font-weight: 300;
	line-height: 1;
	padding: $s-size;
	text-decoration: none;
}

.btn--link {
	background: none;
}

.btn--secondary-inline {
	background: #888;
	position: relative;
	top: -$l-font-size - (2 * $s-size) - $m-size;
	float: right;
}
