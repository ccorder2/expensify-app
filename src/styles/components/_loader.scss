.loader {
	align-items: center;
	display: flex;
	height: 100vh;
	width: 100vw;
	justify-content: center;
}

.loader__img {
	height: 6rem;
	width: 6rem;
}
