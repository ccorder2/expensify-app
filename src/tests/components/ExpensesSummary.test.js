import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should correctly render expenses summary with 1 expense', () => {
	const wrapper = shallow(
		<ExpenseListItem expenseCount={1} expensesTotal={235} />
	);
	expect(wrapper).toMatchSnapshot();
});

test('should correctly render expenses summary with multiple expense', () => {
	const wrapper = shallow(
		<ExpenseListItem expenseCount={23} expensesTotal={48625487} />
	);
	expect(wrapper).toMatchSnapshot();
});
