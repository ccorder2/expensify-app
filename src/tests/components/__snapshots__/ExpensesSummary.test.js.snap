// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render expenses summary with 1 expense 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/undefined"
  >
    <h3 />
  </Link>
  <p>
    $0.00
    -
    December 31st, 1969
  </p>
</div>
`;

exports[`should correctly render expenses summary with multiple expense 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/undefined"
  >
    <h3 />
  </Link>
  <p>
    $0.00
    -
    December 31st, 1969
  </p>
</div>
`;
